vj4/db.py

```diff
@@ -1,29 +1,36 @@
 import aiomongo
 import functools
 import urllib.parse
+from yarl import URL
```

Comment:
`import yarl`

The naming convention of this repository doesn't allow importing classes.